### PR TITLE
add icons assets and define RFIcons enum

### PR DESCRIPTION
**Pull request by roblack**
## Checklist
_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](https://github.com/RocketFanOrg/RocketFanApp/master/CONTRIBUTING.md) doc
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] My changes do not generate new SwiftLint warnings
- [ ] Add GitHub issue #

## Types of changes

What types of changes does your code introduce to RocketFanApp?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Work done
Imported icons as PDFs so scalability is not an issue.
Defined an enum which is used to generate images as Views with the tint already applied to them using the Theme.

## Notes
@squarefrog @RJ-Clegg 
Nothing extra 😉

## Screenshots
image

### CODE REVIEW

**Comment by squarefrog** [comment]
These look good! I bet we could write a unit test for this just to verify they all load.

**Comment by roblack** [comment]
Never did that so really interested how :)

**Comment by roblack** [comment]
Also this lint error is maybe because of .foregroundColor part thing? Since it says "inline_mode: true" ?
Never used linter so quite new to that as well :)

**Comment by squarefrog** [comment]
Let me think about the test overnight. Something like this:

```
func test_images_areInBundle() {
  for image in RFIcons.allCases {
    let image = UIImage(named: image.rawValue)
    XCTAssertNotNil(image, “Failed to load: \(image)”)
  }
}
```

Nah don’t worry about the lint issue, looks like somethings up with our `Dangerfile`

**Comment by roblack** [comment]
Oh nice! That looks/sounds quite clear, tnx!
Daym, I really gotta start looking into tests more. I think i even have a book about it.

**Comment by squarefrog** [comment]
They’re certainly not necessary for something like this, but it’s a neat way to verify all the images have been named correctly without manually trying them all.

File: RocketFan/Utilities/Theme.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import SwiftUI
 
-enum Theme {
+enum Theme: CaseIterable {
```
Comment (squarefrog) [inline RocketFan/Utilities/Theme.swift:3]: We don't need this conformance in this PR 👍

File: RocketFan/Feature/Shared/Views/RFIcons.swift [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import SwiftUI
 
-enum RFIcons: String {
+enum RFIcons: String, CaseIterable {
```
Comment (squarefrog) [inline RocketFan/Feature/Shared/Views/RFIcons.swift:3]: This would be better suited as an extension in `ImageTests.swift`

**Pull request merged**